src/encoded/reports/metadata.py

```diff
@@ -478,6 +480,53 @@ def _generate_rows(self):
                 )
 
 
+class SeriesMetadataReport(MetadataReport):
+
+    DEFAULT_PARAMS = [
+        ('limit', 'all'),
+        ('field', 'related_datasets.files.@id'),
+        ('field', 'related_datasets.files.href'),
+        ('field', 'related_datasets.files.restricted'),
+        ('field', 'related_datasets.files.no_file_available'),
+        ('field', 'related_datasets.files.file_format'),
+        ('field', 'related_datasets.files.file_format_type'),
+        ('field', 'related_datasets.files.preferred_default'),
+        ('field', 'related_datasets.files.status'),
+        ('field', 'related_datasets.files.assembly'),
+        ('field', 'related_datasets.files.related_datasets'),
+    ]
+    FILES_PREFIX = 'related_datasets.files.'
+
+    def _get_column_to_fields_mapping(self):
+        return SERIES_METADATA_COLUMN_TO_FIELDS_MAPPING
+
+    def _should_not_report_file(self, file_):
+        """Only include preferred_default files in addition to existing
+        criteria.
+        """
+        is_file_not_reportable = super()._should_not_report_file(file_)
+        return is_file_not_reportable or not file_.get('preferred_default', False)
```

Comment:
I wonder if it makes sense to just always add `&related_datasets.files.preferred_default=true` to the frontend batch_download link? Putting it here works, but might reduce flexibility in the future if someone was interested in series files that weren't preferred defaults.

Reply:
Thanks! I’ll give that preferred_default idea a little thought, and I’ll add the tests.